apps/judicial-system/backend/src/app/modules/case/internalCase.controller.ts

```diff
@@ -116,7 +116,10 @@ export class InternalCaseController {
     )
   }
 
-  @UseGuards(CaseExistsGuard)
+  @UseGuards(
+    CaseExistsGuard,
+    new CaseTypeGuard([...restrictionCases, ...investigationCases]),
+  )
```

Comment:
This is not part of the current task. I just noticed this a fixed it. The prosecutor is deliverd to the district court in a different way for indictment cases.